rc/extra/autorestore.kak

```diff
@@ -9,32 +9,36 @@ def autorestore-restore-buffer -docstring "Restore the backup for the current fi
 
         ## Find the name of the latest backup created for the buffer that was open
         ## NB. kak doesn't automatically cd to current buffer_dirname.
-        buffer_backup_glob="${buffer_dirname}/.${buffer_basename}.kak.*"
-        backup_path=$(ls -1t ${buffer_backup_glob} 2>/dev/null | head -n 1)
+        buffer_backup_glob=".${buffer_basename}.kak.*"
+#        backup_path=$(ls -1t ${buffer_backup_glob} 2>/dev/null | head -n 1)
+        newer=$(find "${buffer_dirname}" -maxdepth 1 -name "${buffer_backup_glob}" -newer "${kak_buffile}" 2>/dev/null | head -n1)
+        older=$(find "${buffer_dirname}" -maxdepth 1 -name "${buffer_backup_glob}" \! -newer "${kak_buffile}" 2>/dev/null | head -n1)
```

Comment:
Unfortunately, `-maxdepth` isn't [POSIX](http://pubs.opengroup.org/onlinepubs/009695399/utilities/find.html) which is why we usually use `ls -1`, in that case.